bamboo/abort-if-not-pr-or-master.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+source .bamboo_env_vars || true
+if [[ $GIT_PR != true && BRANCH != master ]]; then
+  >&2 echo "******Branch HEAD is not a github PR, and this isn't a redeployment build, skipping step"
```

Comment:
maybe `... and this isn't a master build ...` would be a better explanation of the error?

Reply:
I'm totally guilty of copypasta here.  I'll fix that.

Reply:
49d2bf374